src/services/AuthService.ts

```diff
@@ -0,0 +1,69 @@
+import axios from 'axios';
+import jwt from 'jsonwebtoken';
+import { cache } from 'memory-cache';
+
+var JWT_SECRET = 'my-secret-key';
```

Comment:
This should be in a secrets file and not in github. Also should be an actual secret